content/en/blog/_posts/2018-08-10-introducing-kubebuilder.md

```diff
@@ -0,0 +1,50 @@
+
+With the Kubernetes API model, management logic is embedded directly into an application specific Kubernetes API, e.g. a “MySQL” API.  Users then declaratively manage the application through yaml configuration using tools such as kubectl, just like they do for Kubernetes objects.  This approach is referred to as an Application Controller, also known as an Operator.  Controllers are a powerful technique backing the core Kubernetes APIs that may be used to build many kinds of solutions in addition to Applications; such as Autoscalers, Workload APIs, Configuration APIs, CI/CD systems, and more.
+
+However, while it has been possible for trailblazers to build new Controllers on top of the raw apimachinery, doing so has been a DIY “from scratch” experience, requiring developers to learn low level details about how Kubernetes libraries are written, handwrite boilerplate code, and warp their own solutions for integration testing, RBAC configuration, documentation, etc.  Kubebuilder makes this experience simple and easy by applying the lessons learned from building the core Kubernetes APIs.
+
+## Getting Started Building Application Controllers and Kubernetes APIs
+
+By providing an opinionated and structured solution for creating Controllers and Kubernetes APIs, developers have a working “out of the box” experience that uses the lessons and best practices learned from developing the core Kubernetes APIs. Creating a new “hello world” Controller with kubebuilder is as simple as:
+
+ 1. Create a project with `kubebuilder init`
+ 2. Define a new API with `kubebuilder create api`
+ 3. Build and run the provided main function with `make install & make run`
+
+This will scaffold the API and Controller for users to modify, as well as scaffold passing integration tests, RBAC rules, Dockerfiles, Makefiles, etc.
+After adding their implementation to the project, users build the artifacts to publish their API through:
+
+ 1. Build and push the container image from the provided Dockerfile using `make docker- build / make docker- push`
+ 2. Deploy the API using `make deploy`
+
+Whether you are already a Controller aficionado or just want to learn what the buzz is about, check out the [kubebuilder repo][kubebuilder-repo] or take a look at an example in the [kubebuilder book][kubebuilder-book] to learn about how simple and easy it is to build Controllers.
+
+## Get Involved
+Kubebuilder is a project under [SIG API Machinery][SIG-APIMachinery] and is being actively developed by contributors from many companies such as Google, Red Hat, VMware, Huawei and others. Get involved by giving us feedback through these channels:
+ 
+ - Kubebuilder [chat room on Slack][slack-channel]
+ - SIG [email list][mailing-list]
+ - [Github Issues][open-an-issue]
+ - Send a Pull request in [kubebuilder repo][kubebuilder-repo]
```

Comment:
`Pull request` -> `pull request`

Reply:
`in [kubebuilder repo]` -> `in the [kubebuilder repository]`

Reply:
Done.